### PR TITLE
Add common xf86-video drives + x86-input for tablets

### PR DESCRIPTION
**Pull request by lkocman**
Improve hardware enablement for GPU and input devices.
Helps to address issue #1752

### CODE REVIEW

File: live/src/agama-installer.kiwi [hunk 1]
```diff
@@ -203,7 +210,7 @@
         <package name="ca-certificates-mozilla"/>
     </packages>
     <packages type="bootstrap" profiles="Leap,Leap-PXE">
-		<package name="Leap-release"/>
+        <package name="Leap-release"/>
```
Comment (lkocman) [inline live/src/agama-installer.kiwi:213]: Fixed with retab

File: live/src/agama-installer.kiwi [hunk 2]
```diff
@@ -150,6 +150,13 @@
         <package name="xorg-x11-server"/>
         <package name="xorg-x11-fonts-core"/>
         <package name="xf86-input-libinput" />
+        <package name="xf86-input-wacom" />
```
Comment (lkocman) [inline live/src/agama-installer.kiwi:153]: wacom was taken from installation-images

**Comment by lkocman** [comment]
We will likely need to add more like for virtualbox, raspppi etc. But this can come later.

**Review by imobachgs — APPROVED** [review]

File: live/src/agama-installer.kiwi (hunk 2 above)
Comment (jreidinger) [reply live/src/agama-installer.kiwi:153]: note wacom is used on all archs except s390

File: live/src/agama-installer.kiwi [hunk 3]
```diff
@@ -150,6 +150,13 @@
...
+        <package name="xf86-video-amdgpu" />
+        <package name="xf86-video-fbdev" />
+        <package name="xf86-video-intel" />
+        <package name="xf86-video-nv" />
+        <package name="xf86-video-qxl" />
+        <package name="xf86-video-vesa" />
```
[the remark this answers is not in the document]
Comment (lslezak) [reply live/src/agama-installer.kiwi:159]: Yes, those packages might not even be available on other architectures.

File: live/src/agama-installer.kiwi [hunk 4]
```diff
@@ -150,6 +150,13 @@
         <package name="xorg-x11-server"/>
         <package name="xorg-x11-fonts-core"/>
         <package name="xf86-input-libinput" />
+        <package name="xf86-input-wacom" />
+        <package name="xf86-video-amdgpu" />
```
Comment (lslezak) [inline live/src/agama-installer.kiwi:154]:
```suggestion
        <package name="xf86-video-amdgpu" arch="x86_64"/>
```

File: live/src/agama-installer.kiwi [hunk 5]
```diff
@@ -150,6 +150,13 @@
...
+        <package name="xf86-video-fbdev" />
```
Comment (lslezak) [inline live/src/agama-installer.kiwi:155]:
```suggestion
        <package name="xf86-video-fbdev" arch="x86_64"/>
```

File: live/src/agama-installer.kiwi [hunk 6]
```diff
@@ -150,6 +150,13 @@
...
+        <package name="xf86-video-intel" />
```
Comment (lslezak) [inline live/src/agama-installer.kiwi:156]:
```suggestion
        <package name="xf86-video-intel" arch="x86_64"/>
```

File: live/src/agama-installer.kiwi [hunk 7]
```diff
@@ -150,6 +150,13 @@
...
+        <package name="xf86-video-nv" />
```
Comment (lslezak) [inline live/src/agama-installer.kiwi:157]:
```suggestion
        <package name="xf86-video-nv" arch="x86_64"/>
```

File: live/src/agama-installer.kiwi [hunk 8]
```diff
@@ -150,6 +150,13 @@
...
+        <package name="xf86-video-qxl" />
```
Comment (lslezak) [inline live/src/agama-installer.kiwi:158]:
```suggestion
        <package name="xf86-video-qxl" arch="x86_64"/>
```

File: live/src/agama-installer.kiwi (hunk 3 above)
Comment (lslezak) [inline live/src/agama-installer.kiwi:159]:
```suggestion
        <package name="xf86-video-vesa" arch="x86_64"/>
```

**Review by lslezak — COMMENTED** [review]
Add x86_64 only flag.

Later we can add more architectures and more packages...

File: live/src/agama-installer.kiwi (hunk 3 above)
[the remark this answers is not in the document]
Comment (lkocman) [reply live/src/agama-installer.kiwi]: Fixed! Thank you

**Comment by lkocman** [comment]
Fixed, added x86_64 only to all, except wacom, where I filtered away s390x.

**Review by imobachgs — APPROVED** [review]

**Review by lslezak — APPROVED** [review]

**Pull request merged**